### ISSUE TITLE
Access Past Certificates | Mobile Access

### ISSUE THREAD

**Issue by JessicaMarine1** [GSA/smartpay-training#142]
As a user, I want to access my past certificates from a mobile device, so I have can have access without using a laptop.

Acceptance criteria:

- [ ] User is able access past certificates from a mobile device
  - Note: this story is intended to capture design tweaks, if any, that might be needed for the "access past certificates" flow on mobile

**Comment by weiwang-gsa** [comment]
@melchoyce @A-L33  I finally figured out what causing the mobile issue for the top table.  The 1st table for user information has email address, as we don't break a word as normal practice, the email address can't be wrapped to 2nd line. Which exceed the width that allow it wrapped nicely in the container and make it stick out creating more space on the right.  Please see screen shot for an example with long email address. 

![Image](https://github.com/GSA/smartpay-training/assets/119954777/ba2f8fcd-4109-4d6a-bc91-0131ba1e8f8a)


Possible solutions:
1. break the word for email address cell. 
2. Have new mobile design ( instead of using table layout have all the information in one table row, maybe we can have multiple rows instead? e.g below
Name: xxx
Email:xxx
Agency/Organization: xxxx

Please let me know your thought/suggestions, thanks!

**Comment by A-L33** [comment]
Happy to defer to @melchoyce for solutions but I support whatever is going to be workable and be manageable for effort.

**Comment by melchoyce** [comment]
I think having multiple rows on mobile makes sense 👍 

Would you like me to mock that up, or are you good to go?

**Comment by weiwang-gsa** [comment]
@melchoyce I am fine either way, if not too much trouble, mock up will be better to save some back and forth :)

**Comment by melchoyce** [comment]
Yeah wordbreak makes sense on these emails, I think. Any way to force that column a little wider though, since it seems like (flex?) is prioritizing the rightmost column?

**Comment by weiwang-gsa** [comment]
@melchoyce I tested it, I think the right column occupy the most length is because "Agency/Organization" is treated as one word,  that column name length is the longest and occupy the most of width of the table. I tested with giving different width for each column, even by giving email the most width, but it won't work, unless I break the "Agency/Organization". since this is fixed text, so I tried to add soft break after/ using <wbr>, seems looks better in mobile view. It may not be supported by all browsers though. Chrome is working fine. Let me know if below screen shot looks ok for you. 


![Image](https://github.com/GSA/smartpay-training/assets/119954777/940a25cd-bd28-4214-bf3a-e22fc28698d9)
